src/main/java/io/stargate/sgv2/jsonapi/service/resolver/sort/TableMemorySortClauseResolver.java

```diff
@@ -70,6 +77,20 @@ public WithWarnings<RowSorter> resolve(
       throw new IllegalStateException("Cannot do table vector sorts in memory");
     }
 
+    // in memory sort does not support page state, error out if it is present
```

Comment:
nit: tweak the comment to say because pagination currently only comes from the CQL driver and in memory sort reads all the data